packages/admin/resources/views/components/page.blade.php

```diff
@@ -69,9 +69,11 @@
         >
             @if ($action)
                 @if ($action->isModalCentered())
-                    <x-slot name="heading">
-                        {{ $action->getModalHeading() }}
-                    </x-slot>
+                    @if ($heading = $action->getModalHeading())
+                        <x-tables::modal.heading>
```

Comment:
What happened here? `x-tables::modal` should still be `x-slot`, right? Could you revert all those changes.

Reply:
some copy & paste mistakes. should be fine now